### DISCUSSION TITLE
Example Using Open-Ended Surveys

### DISCUSSION THREAD

**Discussion by reuning** [MaartenGr/BERTopic#2221] [Show and tell]
Not sure if this is helpful for anyone else but I put together a write up on using BERTopic on open-ended survey responses. I did this partially because I wanted to better understand the component parts and thought trying to explain it would help. I have tried to describe how UMAP and HDBSCAN work without getting into the technical details. 

Link to the writeup: https://kevinreuning.com/presentations/pages/bertopic.html

**Comment by PipaFlores** [comment]
Amazing. I am every now and then checking for more resources like this, for pedagogical purposes. I hope you don't mind, but I added it to https://github.com/MaartenGr/BERTopic/discussions/2200
